Microsoft.NET.Build.Containers/ImageConfig.cs

```diff
@@ -12,12 +13,19 @@ internal sealed class ImageConfig
 {
     private readonly JsonObject _config;
     private readonly Dictionary<string, string> _labels;
-    private readonly List<Layer> _newLayers = new();
     private readonly HashSet<Port> _exposedPorts;
     private readonly Dictionary<string, string> _environmentVariables;
     private string? _newWorkingDirectory;
     private (string[] ExecutableArgs, string[]? Args)? _newEntryPoint;
 
+    /// <summary>
```

Comment:
Those look like significant changes - do we need to cover them with tests?

Reply:
That's a good point - I think we could actually do verification on the outputs of the image manifests exactly 1:1 now, instead of potentially having lots of 'noise' in the image configurations.